subprojects/core/src/main/java/org/gradle/api/tasks/TaskExecutionException.java

```diff
@@ -24,7 +24,7 @@
  */
 @Contextual
 public class TaskExecutionException extends DefaultMultiCauseException {
-    private final Task task;
+    private transient final Task task;
```

Comment:
I noticed when running (Gradle via the Tooling API) that sometimes, an attempt is made to serialize this field, hence the `transient` modifier